src/actions/pkg.cr

```diff
@@ -2,26 +2,114 @@ module DppmRestApi::Actions::Pkg
   extend self
   ALL_PKGS = ""
   ONE_PKG  = "/:id"
+
+  def output_config_for(package, to builder)
```

Comment:
Do like in the stdlib https://crystal-lang.org/api/master/JSON/Serializable.html#to_json%28json%3AJSON%3A%3ABuilder%29-instance-method for the typing. (`json : JSON::Builder`)
I suggest a better self-explanatory method name, what does this method do?
I guess `build_package_vars`.

Reply:
> what does this method do?

It *outputs* the *config for package* into *builder*. I don't know how it could be more clear than that. Although, I agree more type restrictions are better.

Reply:
I guess `build_config_response_for`? I suppose *output* isn't the right word.

Reply:
Actually this method won't build a config response _for_ a package, but from, and for the client. I think better to drop this `_for` prefix.

Reply:
Ok.